src/slack/tests/main_test.bal

```diff
@@ -0,0 +1,204 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/lang.'array as arrays;
+import ballerina/lang.'string as strings;
+
+string encodedToken = "eG94cC05NDM1ODEzNjI5OTUtOTQ0OTE3ODc1MDI1LTk3Mjg1NjM0MzgwOS0wZTEwYzc2MGIyNTIzMjkyMjQwODk4ZmNkYWQ2MTBiNg==";
```

Comment:
These credentials should be passed as GitHub secrets.
https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets